### DISCUSSION TITLE
New Release 1.4.0

### DISCUSSION THREAD

**Discussion by sashirestela** [sashirestela/simple-openai#39] [Announcements]
- Added `baseUrl` instead of `urlBase` which has been marked as deprecated (it'll be removed on the next major release).
- Added an example for Chat Vision with local images.

See changes [here](https://github.com/sashirestela/simple-openai/releases/tag/v1.4.0)

**Comment by sashirestela** [comment]
@the-gigi Thanks for your contributions. They have been added to this new release.

**Comment by the-gigi** [comment]
Happy to contribue. This is the best Java library for accessing OpenAI (I tried 4 different ones)
